src/block/count-up/frontend-count-up.js

```diff
@@ -13,9 +13,11 @@ class StackableCountUp {
 	callback = entries => {
 		entries.forEach( entry => {
 			const el = entry.target
+			const duration = el.getAttribute( 'data-duration' ) ? el.getAttribute( 'data-duration' ) * 1000 : 1000
+
```

Comment:
I think it might be good if we shorten this to:
```suggestion
			const duration = el.getAttribute( 'data-duration' ) || 1000
```

I suggested above to use `ms` instead of seconds, so I used `1000` instead of multiplying by 1000